internal/errors/sentinels.go

```diff
@@ -1,44 +1,42 @@
 package errors
 
-// Errors returned from this package may be tested against these errors
-// with errors.Is. Creating new Sentinel type errors like these should be
-// deprecated in favor of the new Err type that includes unique Codes and a
-// Matching function.
+// Errors returned from this package may be tested against these errors with errors.Is.
+// TODO: Deprecate sentinel errors in favor of the new Err type that includes unique ErrorId and Code.
```

Comment:
Note to reviewers:  I plan to deprecate all of these sentinel errors as I work for the refactor of all errors to be the new domain errors.  For now, I added a unique `errorId` to each to explicitly show the error was created via `sentinels.go`